src/target/riscv/riscv.h

```diff
@@ -253,6 +259,8 @@ typedef struct {
 
 	/* Track when we were last asked to do something substantial. */
 	int64_t last_activity;
+
+	yes_no_maybe_t vsew64_supported;
```

Comment:
So this new data member doesn't need an initialization in riscv_info_init because the entire structure it's a part of gets allocated using calloc(), and YNM_MAYBE is the zero representation and what we want it initialized to, and that's what we get.  From a code readability standpoint, I wonder if it's worth initializing vsew64_supported in riscv_info_init, so there's no room for (human) misunderstanding the intent regarding the initial value.  (If OpenOCD community coding standards aren't against that sort of thing).

Reply:
You're right about the logic, and it does seem better to explicitly initialize anyway. Done.